Kitodo/src/main/java/org/kitodo/production/interfaces/activemq/ActiveMQDirector.java

```diff
@@ -49,8 +49,7 @@
  * The class ActiveMQDirector also provides a basic ExceptionListener
  * implementation as required for the connection.
  */
-@WebListener
-public class ActiveMQDirector implements Runnable, ServletContextListener {
```

Comment:
Please adjust JavaDoc as this class does not implement `ServletContextListener` anymore.